_posts/2016-11-20-algoritms4-chapter1.md

```diff
@@ -23,55 +23,72 @@ This chapter will cover these five main topics:
 
```

Comment:
refer to this page http://stackoverflow.com/questions/11948245/markdown-to-create-pages-and-table-of-contents